mono/profiler/proflog.c

```diff
@@ -526,31 +541,36 @@ typedef struct {
 	uint64_t time;
 } MethodInfo;
 
-#ifdef TLS_INIT
-#undef TLS_INIT
-#endif
-
 #ifdef HOST_WIN32
```

Comment:
Those are unrelated changes. And __thread is not going away any time soon either.